bolt/internal/internal.proto

```diff
@@ -11,7 +11,7 @@ message Source {
 	bool Default              = 7;  // Flags an source as the default.
 	string Telegraf           = 8;  // Telegraf is the db telegraf is written to. By default it is "telegraf"
 	bool InsecureSkipVerify   = 9;  // InsecureSkipVerify accepts any certificate from the influx server
-	string MetaURL            = 10; // MetaURL is the connection URL for the meta node.
+	repeated string MetaURL   = 10; // MetaURL is the connection URLs for the meta nodes.
```

Comment:
almost never reuse a number in protobuf.

Go ahead and make this 12 and just comment out 10

Reply:
@goller thanks for the tip. 👍